transport/http/inbound.go

```diff
@@ -201,7 +225,34 @@ func (i *Inbound) start() error {
 		Addr:    i.addr,
 		Handler: httpHandler,
 	})
-	if err := i.server.ListenAndServe(); err != nil {
+
+	addr := i.addr
+	if addr == "" {
+		addr = ":http"
+	}
```

Comment:
what do those 3 lines do?

Reply:
I'm not very sure, taken from: https://github.com/yarpc/yarpc-go/blob/ee0a9dbd2ac833b6c89fc46c9ea5032b5ae811bd/internal/net/httpserver.go#L80-L83

Reply:
Ok found the details, ":http" reads as port 80

https://github.com/golang/go/blob/5531838c046eb1a94b142deeb89d7d8d9be3fc14/src/net/http/server.go#L2585-L2588